gmso/external/convert_mol2.py

```diff
@@ -0,0 +1,175 @@
+"""Convert to and from a TRIPOS mol2 file."""
+# TODO add sources of mol2 files
+import os
+import warnings
+
+import unyt as u
+
+from gmso import Atom, Bond, Box, Topology
+from gmso.core.element import element_by_name, element_by_symbol
+
+
+def from_mol2(
+    filename, site_type="Atom"
+):  # TODO add flags for information to return
+    """Read in a TRIPOS mol2 file format into a gmso topology object.
+
+    Creates a Topology from a mol2 file structure. This will read in the
+    topological structure (sites, angles, and box) information into gmso.
```

Comment:
should this line    "topological structure (sites, angles, and box) information into gmso."
be    "topological structure (sites, bonds, and box) information into gmso."

I did not see an angles determination.

Reply:
You read it right, it should be bonds and not angles.